packages/monorepo-scripts/src/utils/utils.ts

```diff
@@ -48,7 +48,7 @@ export const utils = {
                     };
                     packages.push(pkg);
                 } catch (err) {
-                    utils.log(`Couldn't find a 'package.json' for ${subpackageName}. Skipping.`);
+                    // Couldn't find a 'package.json' for package. Skipping.
```

Comment:
Too noisy?

Reply:
Yeah, too noisy and very unlikely to be an issue.